cloudinit/cmd/main.py

```diff
@@ -72,17 +72,21 @@ def print_exc(msg=""):
     sys.stderr.write("\n")
 
 
-def log_ppid():
-    if util.is_Linux():
+def log_ppid(distro):
+    if distro.is_linux:
         ppid = os.getppid()
-        LOG.info("PID [%s] started cloud-init.", ppid)
+        log = LOG.info
+        extra_message = ""
+        if 1 != ppid and distro.uses_systemd():
+            log = LOG.warning
+            extra_message = ("Not a supported configuration.",)
+        log("PID [%s] started cloud-init. %s", ppid, extra_message)
```

Comment:
Are we intentionally not logging the PID when `not distro.is_linux`?

Reply:
That's what the code did before, but I suppose we should be able to log PPID on non-linux too. I'm curious whether the PPID=1 should hold true on any of the BSDs, and it wouldn't hurt to log it.